package.json

```diff
@@ -39,16 +41,19 @@
     "ember-data": "^2.4.0",
     "ember-disable-proxy-controllers": "^1.0.1",
     "ember-export-application-global": "^1.0.4",
-    "ember-frost-core": ">=0.0.13 <2.0.0",
+    "ember-frost-core": "^0.11.7",
     "ember-load-initializers": "0.5.1",
     "ember-lodash": "0.0.7",
-    "ember-one-way-controls": "0.6.4",
+    "ember-one-way-controls": "0.8.2",
     "ember-resolver": "^2.0.3",
-    "ember-truth-helpers": "^1.2.0",
+    "ember-run-raf": "https://github.com/quincyle/ember-run-raf/tarball/master",
```

Comment:
Looks like he fixed this https://github.com/runspired/ember-run-raf/commit/90153d8db55a4e1f1609a8fea6e03e5741020db8 so we can go back to the published version